apps/api/project.json

```diff
@@ -61,14 +72,22 @@
                 "generate",
                 "build-console"
             ],
-            "command": "bash -c 'CGO_ENABLED=0 go build -o .artifacts/bin/$(go env GOOS)/$(go env GOARCH)/zitadel.local -ldflags=\"-s -w\"'",
+            "command": "bash -c 'EXT=\"\"; if [ \"$(go env GOOS)\" = \"windows\" ]; then EXT=\".exe\"; fi; DATE=$(date \"+%Y-%m-%dT%T%z\" | sed -E \"s/.([0-9]{2})([0-9]{2})$/-\\1:\\2/\"); echo \"Building for $(go env GOOS)-$(go env GOARCH)...\"; CGO_ENABLED=0 go build -o .artifacts/bin/$(go env GOOS)/$(go env GOARCH)/zitadel$EXT -ldflags=\"-s -w -X github.com/zitadel/zitadel/cmd/build.commit=$(git rev-parse --short HEAD) -X github.com/zitadel/zitadel/cmd/build.date=${DATE} -X github.com/zitadel/zitadel/cmd/build.version=${ZITADEL_RELEASE_VERSION:-${DATE}}\"'",
```

Comment:
Can you elaborate what this command should achieve and why we need to care about windows here?

We do not maintain an official windows binary.

Reply:
Ok, then I'll remove it